### PR TITLE
Dev/nojoven/clear old workflows from db

### LINKED ISSUE #24: Clean old workflows
When director has been running for some time, workflows and tasks are starting to pile up inside the database and this is making DB requests slow (along with the UI). Having some default retention for every workflow would prevent this from happening (and being able to set the retention manually in the YAML file would be great, but maybe with a hardcoded max retention limit in director to avoid too high values).

### PR DESCRIPTION
**Pull request by nojoven**
Fix #24

### CODE REVIEW

File: director/__init__.py [hunk 1]
```diff
@@ -80,8 +83,18 @@ def create_app(
     cel_workflows.init_app(app)
     sentry.init_app(app)
 
+    # This list is will be passed to the cleaning function
+    workflows_to_delete = []
```
Comment (ncrocfer) [inline director/__init__.py:87]: This variable is not used, you can remove it.

File: director/__init__.py [hunk 2]
```diff
@@ -14,6 +15,8 @@
 from director.tasks.base import BaseTask
 from director.views import view_bp
 from director.utils import build_celery_schedule
+from director.models.workflows import Workflow
```
Comment (ncrocfer) [inline director/__init__.py:18]: This import is not used

File: docs/docs/guides/clean-workflows.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+Director allows you to permanently delete useless records of your workflows based on a retention value that is a positive integer.
+For example, you can use it to keep only the 10 last records of a workflow every day at midnight.
+
+This feature leverages [celery beat](https://docs.celeryproject.org/en/stable/userguide/periodic-tasks.html#crontab-schedules).
+
+# Prerequisites
+
+This feature requires that you start a Celery Beat scheduler.
+
+You can find more details about the periodic workflows commands in [the build guide](https://ovh.github.io/celery-director/guides/build-workflows/).
```
Comment (ncrocfer) [inline docs/docs/guides/clean-workflows.md:12]: Please use relative path.

File: docs/docs/guides/clean-workflows.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+          - RANDOM
+          - RANDOM
+    - ADD
+
+```
+
+
+# How it works
+
+For each workflow that is present in your *workflows.yml*, the cleaning process will:
```
Comment (ncrocfer) [inline docs/docs/guides/clean-workflows.md:57]: Please don't explain how your code works but how the user can use this feature ^^

**Review by ncrocfer — CHANGES_REQUESTED** [review]

**Review by ismailrei — APPROVED** [review]

**Review by maximecaruchet — APPROVED** [review]

**Review by ncrocfer — APPROVED** [review]

**Pull request merged**